scheds/rust/scx_rusty/src/bpf/main.bpf.c

```diff
@@ -605,6 +613,49 @@ s32 BPF_STRUCT_OPS(rusty_select_cpu, struct task_struct *p, s32 prev_cpu,
 			}
 		}
 	}
+	
+	// the task might also be forked/execd so check for wake up only
+	if (wake_flags & SCX_WAKE_TTWU) {
+		cpu = scx_bpf_pick_idle_cpu((const struct cpumask *)p_cpumask, 0);
```

Comment:
Do we need to this again given that we're here iff the previous attempts have failed?

Reply:
I still see this code path getting hit in my bpf_trace_printk output

Reply:
It's something inherently racy but that doesn't mean we need to keep checking the same condition multiple times. If control reaches this point, it should be safe to consider that we should preempt if we can.